studio/components/interfaces/Database/Extensions/EnableExtensionModal.tsx

```diff
@@ -44,7 +50,7 @@ const EnableExtensionModal: FC<Props> = ({ visible, extension, onCancel }) => {
     return () => {
       cancel = true
     }
-  }, [visible])
+  }, [visible, extension.name])
```

Comment:
should we add `extension.name` to the `if (visible)` check then?

Reply:
actually - or should we just convert this to an RQ?

Reply:
It should be a react-query, but we can do that in a follow up PR, for now the logic is okay!